components/prism-latte.js

```diff
@@ -0,0 +1,178 @@
+/* TODO
+	Add support for the n:syntax attribute macro
+	Add support for string interpolation using {$advancedSyntax} within macros
+*/
+
+(function(Prism) {
+
+	var string_interpolation = {
+		pattern: /{\$(?:{(?:{[^{}]+}|[^{}]+)}|[^{}])+}|(^|[^\\{])\$+(?:\w+(?:\[.+?]|->\w+)*)/,
+		lookbehind: true,
+		inside: {
+			rest: Prism.languages.php
+		}
+	};
+
+	var inside = {
+		'string': [
+			{
+				pattern: /'(?:\\[\s\S]|[^\\'])*'/,
+				greedy: true
+			},
+			{
+				pattern: /"(?:\\[\s\S]|[^\\"])*"/,
+				greedy: true,
+				inside: {
+					'interpolation': string_interpolation
+				}
+			}
+		],
+		'number': /\b0x[\dA-Fa-f]+|(?:\b\d+\.?\d*|\B\.\d+)(?:[Ee][-+]?\d+)?/,
+		'variable': [
+			/\$(?!\d)\w+/,
+			{
+				pattern: /(->)(?!\d)\w+/,
+				lookbehind: true
+			},
+			{
+				pattern: /(\[)(?!\d)\w+(?=\])/,
+				lookbehind: true
+			}
+		],
+		'filter': {
+			pattern: /((?:^|[^|])\|\s*)(?!\d)\w+/,
+			lookbehind: true,
+			alias: 'function'
+		},
+		'function': /\b(?!\d)\w+(?=\s*\()/,
+		'keyword': [
+			{
+				pattern: /\b(?:true|false)\b/,
```

Comment:
Why is this a keyword and a boolean? Almost all other languages (including PHP) have a `boolean` pattern and that's it, no `keyword`.